modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/simple/CurveEncoder.java

```diff
@@ -60,8 +60,12 @@ protected CurveEncoder(Encoder e, String gmlPrefix, String gmlUri) {
         this.arcString = ARC_STRING.derive(gmlPrefix, gmlUri);
     }
 
-    public void encode(LineString geometry, AttributesImpl atts, GMLWriter handler)
+    public void encode(LineString geometry, AttributesImpl atts, GMLWriter handler, String gmlId)
```

Comment:
Public API change without backwards compatibility. Please add a method
```
public void encode(LineString geometry, AttributesImpl atts, GMLWriter handler)
```
that calls
```
encode(geometry, atts, handler, null)
```

Reply:
Oh, I see, you did add the backwards-compatible `encode`, pulled up into the base class.